plugins/modules/cloud/lxd/lxd_profile.py

```diff
@@ -179,9 +204,7 @@
   type: list
   sample: '["create"]'
 '''
-
 import os
```

Comment:
I think you should keep the whitespace above this import, and probably also the one below it.

Reply:
Ok